src/widget/wnumberduration.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "preferences/dialog/dlgprefdeck.h"
+#include "skin/legacy/skincontext.h"
+#include "wnumber.h"
+
+class ControlProxy;
+
+class WNumberDuration : public WNumber {
```

Comment:
Could you please add some doc-string to document what this component is used for?